packages/cli/src/lib/handleWorkingDirectory.ts

```diff
@@ -1,3 +1,12 @@
+import { join } from 'path';
+import { setConfiguration } from '../rpc/configuration';
+
 export function handleWorkingDirectory(directory?: string) {
   if (directory) process.chdir(directory);
 }
+
+export function configureRpcDirectories(directory: string | string[]) {
```

Comment:
I know I have said this a lot; this should be called from the `index` command also.